nginx/assets-server-section.conf

```diff
@@ -0,0 +1,7 @@
+location ~ \.(css|js|png|svg) {
+    add_header Cache-Control "max-age=3600, public";
+    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
+    add_header X-Content-Type-Options nosniff always;
+    add_header Content-Security-Policy "default-src 'none'; frame-ancestors 'self'; form-action 'self'; base-uri 'self'; manifest-src 'self'; script-src 'self' 'unsafe-inline' *.service.mod.gov.uk *.dev.service.mod.gov.uk; frame-src 'self' *.service.mod.gov.uk *.dev.service.mod.gov.uk; style-src 'self'; connect-src 'self' *.service.mod.gov.uk *.dev.service.mod.gov.uk; img-src 'self';" always;
+    add_header Permissions-Policy "accelerometer=(self), autoplay=(self), camera=(self), encrypted-media=(self), fullscreen=(self), geolocation=(self), gyroscope=(self), magnetometer=(self), midi=(self), payment=(self), picture-in-picture=(self), sync-xhr=(self), usb=(self)" always;
```

Comment:
Unsure if I need these for css,js,png and svg

Reply:
Might be nice to factor the common bit out into e.g. `nginx/include/common-server-section.conf` and then do e.g.

```
include '/opt/app-root/nginx/include/common-server-section.conf';
```

in this and `nginx/owasp-server-section.conf`.

(The Dockerfile will need to be updated to copy that file e.g. `COPY nginx /opt/app-root/nginx`.)